include/GafferSceneUI/Private/Inspector.h

```diff
@@ -178,8 +178,10 @@ class GAFFERSCENEUI_API Inspector : public IECore::RefCounted, public Gaffer::Si
 		virtual EditFunctionOrFailure editFunction( Gaffer::EditScope *editScope, const GafferScene::SceneAlgo::History *history ) const;
 
 		/// Can be implemented by derived classes to provide a fallback value for the inspection,
-		/// used when no value is returned from `value()`.
-		virtual IECore::ConstObjectPtr fallbackValue( const GafferScene::SceneAlgo::History *history ) const;
+		/// used when no value is returned from `value()`. Optionally, `description` may be
+		/// assigned a description to be shown to the user. Typically, this description would
+		/// be used to disambiguate the source of the fallback value.
+		virtual IECore::ConstObjectPtr fallbackValue( const GafferScene::SceneAlgo::History *history, std::string &description ) const;
```

Comment:
Let's move this under `value()` or `source()` as well, so the same stuff is grouped consistently everywhere.

Reply:
Moved under `value()` in cbb151a.